tests/ops/functions/test_equal.py

```diff
@@ -107,6 +108,66 @@
 )
 
 
+PARAMETRIZED_MEASUREMENTS = [
+    qml.sample(qml.PauliY(0)),
+    qml.sample(wires=0),
+    qml.sample(),
+    qml.counts(qml.PauliZ(0)),
+    qml.counts(wires=[0, 1]),
+    qml.counts(wires=[1, 0]),
+    qml.counts(),
+    qml.density_matrix(wires=0),
+    qml.density_matrix(wires=1),
+    qml.var(qml.PauliY(1)),
+    qml.var(qml.PauliY(0)),
+    qml.expval(qml.PauliX(0)),
+    qml.expval(qml.PauliX(1)),
+    qml.probs(wires=1),
+    qml.probs(wires=0),
+    qml.probs(qml.PauliZ(0)),
+    qml.probs(qml.PauliZ(1)),
+    qml.state(),
+    qml.vn_entropy(wires=0),
+    qml.vn_entropy(wires=0, log_base=np.e),
```

Comment:
For these two, however, I was expecting the test to fail. 
According to the definition of `vn_entropy` [(here)](https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/measurements.py#L1265), if log_base not specified, natural log (i believe is np.e) is used. When I looked at the way `vn_entropy` is defined, I do not see where the natural log being used. Would you be able to provide some clarity? 
Which value, when passed in as `log_base`, would cause these two measurements to be equal?

Reply:
The docstring is wrong:
```pycon
>>> meas = qml.vn_entropy(wires=0)
>>> meas.log_base

```

Thus `qml.equal(qml.vn_entropy(wires=0), qml.vn_entropy(wires=0, log_base=np.e))` should return `False`.

Reply:
Let's leave the docstring as it is, because maybe there is some logic outside of the `MeasurementProcess` class that converts this value to the natural log.

Reply:
Sounds good. Speaking of docstrings, the docstring for `Measurement` class does not include `log_base` or `id` (see [here](https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/measurements.py#L126)), so it would be nice to add an issue for updating it along with `vn_entropy`'s docstring if an issue is not already open.